packages/core/src/sync/sync-api.ts

```diff
@@ -51,22 +52,37 @@ export type SyncConfig = {
 
 // TODO (CDB-2106): move to SyncStatus Class
 export interface ActiveSyncStatus {
+  // The block the sync starts at
+  startBlock: number
+  // The block the sync is currently processing
   currentBlock: number
+  // The block the sync will end on
   endBlock: number
+  // Models that are being synced
   models: Array<string>
+  // Date when the sync was requested
   created: Date
+  // Date when the sync started
```

Comment:
could there ever be more than a few milliseconds difference between `created` and `startTime`?

Reply:
We can only have one History Sync happening at a time (this may change), we currently can only have one history sync being worked on at once (that may change in the future). But if a model is added, a history sync job is queued up for later. We do not error if a user tries to index + sync a model while a sync is going on. I can change that

Reply:
ah interesting.  Yeah I thought we were going to error if a user tried to add a model while a sync was ongoing.  Seems like more of a product question though.  @3avi @michaelsena